### ISSUE TITLE
L1/L2 regularization not found in the code

### ISSUE THREAD

**Issue by aretor** [brain-research/realistic-ssl-evaluation#25]
Hi guys,
In the paper it is stated that you use L1 and L2 regularizations, however I haven't found them in the code. Am I missing something?
Thanks in advance

**Comment by Jongchan** [comment]
I also cannot find any regularization in the code. And running the fully supervised baseline (CIFAR10, 4000 labeled) quickly goes up to overfitting with test accuracy around 0.70~0.73.
The exact numbers should wait for several hours, but since the training seems to be overfit (avg loss scale around 10^-4, avg training accuracy > 0.98)

**Comment by Jongchan** [comment]
I couldn't find any regularization codes, too. 
However, I was able to achieve the reported number in the paper (CIFAR100 in WRN28-2 with 4,000 labeled samples). 
The paper says **20.26 +- 0.38**, and I've got **19.54%**. 
And, I've got VAT+EM **12.75%** (13.13+-0.39 on paper).

**Comment by avital** [comment]
Apologies for the late reply. That's right, there's no L1 regularization. We should fix the paper.

**Comment by avital** [comment]
I submitted a new version to arXiv that no longer talks about L1 and L2 regularization.
